app.py

```diff
@@ -144,6 +144,7 @@ def index():
                 features[name] = value
         recipe = json.dumps(variables, ensure_ascii=False) # Dumping the generator-packages into a JSON array
         feature = json.dumps(features, ensure_ascii=False) # Dumping the chosen features into a JSON objects
+        INSTALL = request.form.getlist('INSTALL')
```

Comment:
@rpotter12 this is already implemented here: https://github.com/fossasia/meilix-generator/blob/09fcf76c52b8dd067c6ad2ad9a9c7aa104252cc6/app.py#L141
No need of the pr, as the response on checkboxes is already implemented
Try triggering a build using your deployment